server/app/services/applicant/question/PhoneQuestion.java

```diff
@@ -123,12 +120,7 @@ public String getAnswerString() {
               getPhoneNumberValue().orElse(""), getCountryCodeValue().orElse(""));
       return PHONE_NUMBER_UTIL.format(phoneNumber, PhoneNumberUtil.PhoneNumberFormat.INTERNATIONAL);
     } catch (NumberParseException e) {
-      logger.error(
-          "Unable to retrieve or parse phone number "
-              + getPhoneNumberValue().orElse("")
-              + "for country_code "
-              + getCountryCodeValue().orElse(""));
+      return "-";
```

Comment:
Blocking: Can you add a javadoc comment that says `Returns formatted phone number or defaults to "-" if formatting fails. Format example: __ADD_EXAMPLE_HERE__` and replace that placeholder with an actual example of the formatted number.

Reply:
done